subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheUnsupportedTypesIntegrationTest.groovy

```diff
@@ -277,8 +272,6 @@ class ConfigurationCacheUnsupportedTypesIntegrationTest extends AbstractConfigur
         // Dependency Resolution Types
         DefaultConfigurationContainer         | ConfigurationContainer         | "project.configurations"
         DefaultResolutionStrategy             | ResolutionStrategy             | "project.configurations.maybeCreate('some').resolutionStrategy"
-        ErrorHandlingResolvedConfiguration    | ResolvedConfiguration          | "project.configurations.maybeCreate('some').resolvedConfiguration"
```

Comment:
I would expect that we still want to verify that these types are unsupported for CC

Reply:
I disagree here.  If we're going to stop using the type, why bother to validate that it continues to not work?

Reply:
Because it still exists. If we extend this logic further, why bother testing `ResolvedConfiguration` anywhere? 

@gradle/bt-configuration-cache Thoughts?

Reply:
I'm not saying don't test it anywhere, there are several test classes specific to this type still present.  I'm saying this specific _negative test_, that verifies it continues to _not_ work is unnecessary if we've decided to (internally) deprecate it and never plan on making it work.

Reply:
I am not sure what the motivations of such a negative test are, other than that if this type suddenly started working with CC that would mean something is wrong with CC.

Reply:
This is testing that we produce a nice error message.  In `UnsupportedTypesCodecs.kt`, we have a list of types that we want to explicitly say are unsupported.

I don't think we want any of these changes.